### ISSUE TITLE
[Google Workspace] Multi-account support for us students!

### ISSUE THREAD

**Issue by jormbee** [raycast/extensions#4039]
### Extension

https://www.raycast.com/raycast/google-workspace

### Description

Basically the title, it'd be great to have multi-account support for us students with both personal and school Google Drive accounts, as opposed to having to pick one!

**Comment by thomaslombart** [comment]
Multi-account support is a commonly required feature that's not possible right now, so I'm afraid it won't be possible to do that for the Google Workspace extension. You can follow the status of this feature request here: https://github.com/raycast/extensions/issues/269.

**Comment by udiNur** [comment]
any updates

**Comment by m4uli** [comment]
I also need this feature: We use 1Password at our company, but I also use it privately and can add both accounts in 1Password, which is why it works very well with the native app.

**Comment by thomaslombart** [comment]
Yes, you can view it here: https://github.com/raycast/extensions/issues/269.

**Comment by thedanbaird** [comment]
Could really use this feature, any news?
